package.json

```diff
@@ -85,8 +85,17 @@
     "check:bundlesize": "bundlesize -f dist/sweetalert2.all.min.js -s 15kB",
     "check:tests": "testem ci",
     "check:ts": "tsc sweetalert2.d.ts",
-    "release": "node release"
+    "release": "node release",
+    "postinstall": "opencollective postinstall"
   },
   "bugs": "https://github.com/sweetalert2/sweetalert2/issues",
-  "license": "MIT"
-}
+  "license": "MIT",
+  "dependencies": {
+    "opencollective": "^1.0.3"
```

Comment:
this should be in devDeps I believe

Reply:
Actually this is a dependency and not a dev dependency. It makes a network request. Two alternatives:

One user made a lighter version of the post install: https://github.com/WebReflection/lightercollective
You can also remove the dependency altogether: https://github.com/remy/nodemon/blob/master/package.json#L42